src/components/Presentation/Dropzone.vue

```diff
@@ -0,0 +1,94 @@
+<template lang="pug">
+  .dropzone-wrapper(
+      v-if="!files.length", :class="dropzoneClass", @dragenter="dropzoneClass = 'animate'",
+      @dragleave="dropzoneClass = null", @drop="dropzoneClass = null")
+    v-layout.dropzone-message(column, justify-center, align-center, fill-height)
+      v-icon(size="50px") $vuetify.icons.fileUpload
+      .title.mt-3 {{ message }}
+    input.file-input(type="file", :multiple="multiple", @change="$emit('change', $event)")
```

Comment:
Should this component have `v-model` support for the files? Seems like it's always going to be two-way binding from some parent component.

Reply:
Yes, I agree. I'll add this.

Reply:
Done.  Added an immutable splice function.

Reply:
I see you added it to the FileUploadList, but I was wondering if it makes sense to add it to the Dropzone component as well? (Since it's an input)

Reply:
Oops, yeah that does make sense.